test/extended/apiserverauth/api.go

```diff
@@ -0,0 +1,40 @@
+package apiserverauth
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/tidwall/gjson"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-API_Server]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		kubectl = exutil.NewCLIWithoutNamespace("default")
+	)
+
+	g.It("Author:knarra-Medium-1001-[Smoke] Checking apiserver version should display", func() {
+		g.By("# Check the apiserver version gitVersion should exist")
+		out, err := kubectl.Run("version").Args("-o", "json").Output()
+		o.Expect(gjson.Get(out, `serverVersion.gitVersion`)).Should(o.ContainSubstring("kcp-v"))
```

Comment:
check here should be to make sure that there are no dirty flags, also kcp version should not show 0.0.0

Reply:
Got you. But accually maybe we don't have a better config for this checkpoint unless we also use the branches like DEV.

Reply:
@Phaow why do we need branches for this ? i did not really quite understand

Reply:
E.g. We need to check details about the main version and sub version. like 0.6.x. 0.7.x different branchs could use different config check this. If all use the same branch we just check the version not equal 0.0.0 maybe not serious. Or currently we could use a ENV var instead, or just simply check `not equal 0.0.0`.

Reply:
@Phaow let us make sure it is not equal to 0.0.0 first then later we could evolve it may be ? WDYT ?

Reply:
Sure. I agree.